### ISSUE TITLE
New Iceland Phone Numbers Are Not Recognized

### ISSUE THREAD

**Issue by shurec** [googlei18n/libphonenumber#862]
Iceland 539 area code is not recognized .
For example, +354 539 0300 and +354 539 0317

Phone Number entered: 354 539 0300
defaultCountry entered: IS
Language entered: en
Parsing Result
country_code	354
national_number	5390300
extension	 
country_code_source	FROM_NUMBER_WITHOUT_PLUS_SIGN
italian_leading_zero	false
raw_input	354 539 0300

Validation Results
Result from isPossibleNumber()	true
Result from isValidNumber()	false
Phone Number region	IS
Result from getNumberType()	UNKNOWN
Formatting Results
E164 format	invalid
Original format	354 539 0300
National format	539 0300
International format	invalid
Out-of-country format from US	invalid
Out-of-country format from CH	invalid

**Comment by mihaelacr-google** [comment]
Thanks for your report!

The prefix verifies as valid by checking up the rages defined by authorities [here]
(http://www.pfs.is/english/telecom-affairs/numbering).  We will change the metadata to reflect this in our next release.

I would also like to friendly remind you that we appreciate your contributions and it would be easier for us if you include the official documentation in your request (most of the time you can start with looking at the ITU official documentation, this is how I got the above link). Thanks!

**Comment by oddurk** [comment]
I have seen the same for numbers starting with:
785

**Comment by lararennie** [comment]
Seems this has been fixed, sorry for not updating earlier!

**Comment by nhuhoan** [comment]
I have seen the same for numbers starting with:
763

**Comment by lararennie** [comment]
Please open a new issue and see our contribution guidelines for the information we require before we can fix this. Thanks!

**Comment by lararennie** [comment]
(https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md)
